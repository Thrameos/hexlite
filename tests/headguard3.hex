d(1). d(2).
f(1).
q(X) : not f(Y) :- d(X).
