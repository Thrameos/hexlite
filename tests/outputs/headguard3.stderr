1 grep -q "unsafe variables: Y"
